test/unit/lib/cgroups/tools_test.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'cgroups/tools'
+
+class ToolsTest < ActiveSupport::TestCase
+
+  test 'calculate available_cpus for cgroups v1' do
+    File.stubs(:exist?).returns(false)
+
+    # 1024 shares is equivalent to 1 cpu
+    File.stubs(:read).returns('2048')
+    assert_equal 2, Cgroups::Tools.available_cpus
```

Comment:
a nitpick, perhaps use another value so that it doesn't match the default of `2`. But everything is great!